rt_gene/scripts/estimate_blink.py

```diff
@@ -55,14 +57,16 @@ def callback(self, msg):
 
         probs, _ = self.predict(left_eyes, right_eyes)
 
+        self.publish_msg(msg.header.stamp, subjects, probs)
+
         if self.viz:
             blink_image_list = []
             for subject, p in zip(subjects.values(), probs):
                 resized_face = cv2.resize(subject.face, dsize=(224, 224), interpolation=cv2.INTER_CUBIC)
                 blink_image_list.append(self.overlay_prediction_over_img(resized_face, p))
 
             if len(blink_image_list) > 0:
-                blink_viz_img = self.cv_bridge.cv2_to_imgmsg(np.hstack(blink_image_list))
+                blink_viz_img = self.cv_bridge.cv2_to_imgmsg(np.hstack(blink_image_list), encoding="bgr8")
```

Comment:
I'm assuming you added that because the colour coding is incorrect otherwise? @Twarz didn't we check this?

Reply:
Unfortunately it wasn't working for me - the error was that 8UC3 didnt't have a matching type in both RVIZ and rqt_image_viewer.... BGR8 solved that issue

Reply:
Fair enough, sounds good thanks